_targets/build.project.ia32-generic

```diff
@@ -48,9 +48,13 @@ b_build_target() {
 	ln -f "$PREFIX_ROOTFS/bin/"{psh,reboot}
 	ln -f "$PREFIX_ROOTFS/bin/"{psh,top}
 
+	if ! [[ ${SYSPAGE} ]]; then
```

Comment:
should have default value (the same as `CONSOLE`), eventually `if [ -z "$SYSPAGE" ];`